conf/ccga_dx.config

```diff
@@ -13,6 +13,7 @@ params {
 
 singularity {
   enabled = true
+  runOoptions = "-B /mnt"
```

Comment:
This looks like a typo! 
```suggestion
  runOptions = "-B /mnt"
```

Reply:
Ah...oh well, should be fixed in most recent commit.